newIDE/app/src/Utils/BlobDownloadUrlHolder.js

```diff
@@ -8,27 +8,31 @@ type Props = {
 
 export const BlobDownloadUrlHolder = ({ blob, children }: Props) => {
   const [blobDownloadUrl, setBlobDownloadUrl] = React.useState('');
+  const [stateBlob, setStateBlob] = React.useState(null);
```

Comment:
```suggestion
  const [stateBlob, setStateBlob] = React.useState<?Blob>(null);
```

Reply:
Let's maybe rename this to `currentBlob`/`setCurrentBlob`?
To show that it's holding the "current value"?